test/WebSites/ControllersFromServicesClassLibrary/ResourcesController.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace ControllersFromServicesClassLibrary
 {
+    [Controller]
```

Comment:
Why is this needed now? Is this because this class library doesn't reference MVC?